server/helpers/custom-validators/video-ownership.ts

```diff
@@ -0,0 +1,42 @@
+import { Response } from 'express'
+import * as validator from 'validator'
+import { VideoChangeOwnershipModel } from '../../models/video/video-change-ownership'
+import { UserModel } from '../../models/account/user'
+
+export async function doesChangeVideoOwnershipExist (id: string, res: Response): Promise<boolean> {
+  const videoChangeOwnership = await loadVideoChangeOwnership(id)
+
+  if (!videoChangeOwnership) {
+    res.status(404)
+      .json({ error: 'Video change ownership not found' })
+      .end()
+
+    return false
+  }
+
+  res.locals.videoChangeOwnership = videoChangeOwnership
+  return true
+}
+
+async function loadVideoChangeOwnership (id: string): Promise<VideoChangeOwnershipModel | undefined> {
+  if (validator.isInt(id)) {
+    return VideoChangeOwnershipModel.load(parseInt(id, 10))
+  } else {
+    return undefined
+  }
+}
+
+export function checkUserCanTerminateOwnershipChange (
+  user: UserModel,
+  videoChangeOwnership: VideoChangeOwnershipModel,
+  res: Response
+): boolean {
+  if (videoChangeOwnership.NextOwner.userId === user.Account.userId) {
+    return true
+  } else {
```

Comment:
Don't need this `else` because you `return true`

Reply:
Hmm… to better understand the algorithm, we should keep the else part. It gives a quick indication of the function structure (code pattern).

Same for other comments below.

Reply:
The else is unneeded, it's implicit because the `return` is just above. It adds 2 more lines and makes it more difficult to see the important information: the `return`.

They are two different visions but to keep consistency with the rest of the project, please remove it